indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,29 @@
-{% extends 'layout/full_content_base.html' %}
+{% extends 'layout/base.html' %}
+{% from 'events/tracks/_track_list.html' import render_track_list %}
 
 {% block title %}
     {%- trans %}Programme{% endtrans -%}
 {% endblock %}
 
-{% block content %}
-    {% for track in tracks %}
-        <div>
-            {{ track.title }}
-        </div>
-    {% endfor %}
+{% block description %}
+    {%- trans -%}
+        Here you can find a list of tracks for this event.
```

Comment:
This description is not very useful. Better explain what tracks are used for.

> Tracks represent the subject matter of the conference. They represent both the basis for the Scientific Programme and a taxonomy that can be using to classify submissions during the Call for Abstracts process.

Something similar to this (best to discuss on IRC on the wording)